src/main/resources/application.properties

```diff
@@ -0,0 +1,25 @@
+#spring.h2.console.enabled=true
```

Comment:
Please remove this commented code. Keep in mind to not push commented code into main branch